ethclient/ethclient.go

```diff
@@ -102,9 +102,10 @@ func (ec *Client) PeerCount(ctx context.Context) (uint64, error) {
 }
 
 type rpcBlock struct {
-	Hash         common.Hash      `json:"hash"`
-	Transactions []rpcTransaction `json:"transactions"`
-	UncleHashes  []common.Hash    `json:"uncles"`
+	Hash         common.Hash         `json:"hash"`
+	Transactions []rpcTransaction    `json:"transactions"`
+	UncleHashes  []common.Hash       `json:"uncles"`
+	Withdrawals  []*types.Withdrawal `json:"withdrawals"`
```

Comment:
```suggestion
	Withdrawals  []*types.Withdrawal `json:"withdrawals,omitempty"`
```
I think omitting the withdrawals if empty is more in line with how we marshal other stuff to json

Reply:
I was not sure if that object gets serialized to json at some point (as the library is mainly used for parsing the json rpc api responses into go types) but I've pushed a fix anyway. Thanks for the suggestion.